lib/Fmt_ast.ml

```diff
@@ -4239,9 +4246,8 @@ and fmt_structure_item c ~last:last_item ?ext {ctx; ast= si} =
             let epi =
               match c.conf.let_binding_spacing with
               | `Compact -> None
-              | `Sparse ->
-                  let force_fit_if = last && last_grp && last_item in
-                  Some (fits_breaks ~force_fit_if "" "\n")
+              | `Sparse when last && last_grp && last_item -> None
```

Comment:
with Sparse, if the condition is true, this used to return `Some (fits_breaks ~force_fit_if:false "" "\n")`, but now returns `None`. Is this equivalent? (just checking)

Reply:
When `last && last_grp & &last_itm` was true it just returned `Some (fmt "")`(since the force_fit mode means we just format the fits string which is empty) that was used for epi, so we can just have None as epi instead.